src/store/api/ContactUsAPI.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import requestConfig from "./config";
+import { IContactUs } from "../../interfaces";
+const BASE_URL = process.env.REACT_APP_API_ENDPOINT as string;
+
+class ContactUsAPI{
+    
+    static createContactUs(contactData: IContactUs): Promise<IContactUs> {
+        return axios.get(`${BASE_URL}/admin/contactus`, contactData);
+    }
+
+    static getselectedContactUsId(contactUsId : string): Promise <IContactUs> {
+        return axios.get(`${BASE_URL}/admin/contactus/${contactUsId}`, requestConfig);
+    }
+
+    static getContactsUs(): Promise <IContactUs[]>{
+        return axios.get(`${BASE_URL}/admin/contactus/`, requestConfig);
+    }
+
+    static deleteContactUs(contactUsId : string): Promise <IContactUs> {
+        return axios.put(`${BASE_URL}/admin/contactus/delete/${contactUsId}`, requestConfig);
+    }
+    static deleteContactsUs() : Promise <IContactUs[]> {
+        return axios.put(`${BASE_URL}/admin/contactus/delete/`, requestConfig);
```

Comment:
I think this should be a `get` method. Because here we get all the deleted contact us from the database. **Change it to `get` method**. And also this method will throw an error because if we use the `put` method, there are 3 parameters that need to be passed (route: string, data: Object, headers).